src/ReSharper.Structured.Logging/ReSharper.Structured.Logging.Rider.csproj

```
@@ -10,7 +10,10 @@
     <RootNamespace>ReSharper.Structured.Logging</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="JetBrains.Rider.SDK" Version="2020.3.2" />
+    <PackageReference Include="JetBrains.Rider.SDK" Version="2021.1.0-eap06">
```

The [non-EAP](https://www.nuget.org/packages/JetBrains.Rider.SDK/2021.1.0) version is now available.
```suggestion
    <PackageReference Include="JetBrains.Rider.SDK" Version="2021.1.0">
```